common/viewer.cpp

```diff
@@ -1783,13 +2118,24 @@ namespace rs2
             last_texture = texture;
         }
 
-        glViewport(static_cast<GLint>(viewer_rect.x), static_cast<GLint>(viewer_rect.y),
-            static_cast<GLsizei>(viewer_rect.w), static_cast<GLsizei>(viewer_rect.h));
+        auto bottom_y = win.framebuf_height() - viewer_rect.y - viewer_rect.h;
+        
+        glViewport(static_cast<GLint>(viewer_rect.x), static_cast<GLint>(bottom_y),
+            static_cast<GLsizei>(viewer_rect.w), static_cast<GLsizei>(viewer_rect.h - top_bar_height));
 
         glClearColor(0, 0, 0, 1);
         glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
-        matrix4 perspective_mat = create_perspective_projection_matrix(viewer_rect.w, viewer_rect.h, 60, 0.001f, 100.f);
+        glLoadIdentity();
+
+        glMatrixMode(GL_PROJECTION);
+        glPushMatrix();
+        gluPerspective(45, viewer_rect.w / win.framebuf_height(), 0.001f, 100.0f);
```

Comment:
In case of T265 100 meter won't be enough  imho as you may want to see the whole path